### ISSUE TITLE
Make it possible to tag functions as ignorable

### ISSUE THREAD

**Issue by Keno** [Keno/Gallium.jl#20]

**Comment by ihnorton** [comment]
I'm not sure if this is exactly what you meant by this issue, but I find myself wanting to be able to ignore some intermediate functions when stepping up and down the stack, namely `jl_apply`, `jl_apply_generic` and friends. Those were formerly useful with gdb when they were the only way to get locals, but now they are mostly annoying. :trophy:

**Comment by Keno** [comment]
Yes, navigating only julia frames is on the to do list. The cause of this issue was a request to be able to set certain functions as to-be-ignored when stepping in.
